### ISSUE TITLE
"Unknown location" OpenStreetMap explanation

### ISSUE THREAD

**Issue by harry-wood** [wildtreetech/sentinel2-bot#14]
I saw the tweets because it sometimes says "Unknown location, do you know it? Tell @openstreetmap" e.g. https://twitter.com/Sentinel2Bot/status/785543415547191296
what does this mean? and what are we supposed to be telling OpenStreetMap?

...is my question. I guess the answer might be added to the README here

**Comment by betatim** [comment]
Good point. My idea was that someone might recognise the place in the picture and then take the time to update what ever it is that needs updating in @openstreetmap so that next time the reverse geocoding would work.

From watching images for which it doesn't work it seems most are above the sea, so maybe not that useful.

**Comment by harry-wood** [comment]
Yeah. It sort of sounds like a call to action, but then the action isn't entirely clear. Is there's a higher level in the geocoding result which you could look at? If not, then maybe that's an interesting use-case which could be an issue to raise with the geocoder (are you using nominatim?)
What I mean is...  I guess ideally for this use case, reverse geocoding should be able to say _something_ about every location on the planet, even if it's pretty vague: "atlantic ocean", "arctic circle", "siberia"

**Comment by betatim** [comment]
Yes using nominatim ([ugly code](https://github.com/wildtreetech/sentinel2-bot/blob/master/sentinel2.py#L122-L143)). Using the display name property and shortening it when it doesn't fit in a tweet.

Here some ideas on a better tweet text:
- Unknown location, do you know it? Tell @openstreetmap
- @openstreetmap doesn't know the name of this place, do you? Tell them <link>
- Do you know the name of this place? Tell @openstreetmap
- Do you know the name of this place? Tell @openstreetmap <link>
- Teach @openstreetmap what this place is called: <link>
- Where is this? Make @openstreetmap smarter: <link>
